supabase/functions/download_link/index.ts

```diff
@@ -8,35 +8,50 @@ interface DataDownload {
   storage_provider: string
   user_id?: string
   bucket_id: string
+  api_key?: string
 }
 
 serve(async (event: Request) => {
   if (event.method === 'OPTIONS')
     return sendOptionsRes()
+
   const authorization = event.headers.get('authorization')
-  if (!authorization)
-    return sendRes({ status: 'Cannot find authorization' }, 400)
+  if (!authorization) {
+    return sendRes({ status: 'Cannot find authorization token in header' }, 400)
```

Comment:
please lint the code with `pnpm run lint`

Reply:
Okay, turns out it is a problem that is unrelated to your PR. You do not have automatic linting enabled in your editor. For now do not worry about this. I will create an issue for linting as linting does not trigger for the `supabase` folder and I have automatic linting enabled so I just cough that this is not linted properly

Reply:
This is exactly how I saw it in my head and how my automatic linting (configured based on capgo settings) linted it

![image](https://github.com/Cap-go/capgo/assets/50914789/e531d564-7e39-4a1f-b27b-5192bc40ea7e)